pallets/Polkadex_Ido/src/lib.rs

```diff
@@ -561,7 +534,7 @@ decl_module! {
             let current_block_no = <frame_system::Pallet<T>>::block_number();
             // Ensure investors can only show interest during the voting period of the round
             // Investors who shows interested are randomly selected (whitelisted)  to participate in the round at round start block
-            ensure!(current_block_no < funding_round.start_block, <Error<T>>::NotAllowed);
+            ensure!(current_block_no >= funding_round.start_block, <Error<T>>::NotAllowed);
```

Comment:
Should there not be two checks? and is `<` vs `>` correct?

Reply:
From the last discussions we are using the `show_interest`  also participates the the investor so, must called between on a block `funding_round.start_block` and `funding_round.close_round_block`

Reply:
From the last discussions we are using the `show_interest`  also participates the the investor so, must called between on a block `funding_round.start_block` and `funding_round.close_round_block`

Reply:
Yes so there should be two checks